platform-hub-api/db/migrate/20171012110416_import_kubernetes_tokens.rb

```diff
@@ -0,0 +1,94 @@
+class ImportKubernetesTokens < ActiveRecord::Migration[5.0]
```

Comment:
It's quite possible that the tokens sync job will run during this migration (if we're very unlucky). Wonder if we should somehow turn this off during the migration? E.g. we could turn off the `kubernetes_tokens` feature flag?

Reply:
Sure, we could turn the flag off and switch it back at the end of migration. Though token sync won't change token data so it should have no effect plus it's gonna be rather quick. Alternatively we can take it out of migration and run as standalone.

Reply:
The issue is if the tokens sync job is runs just before the migration kicks in, or halfway through, and then it writes out an incomplete tokens.csv file to S3 – that could screw things up in the cluster, I think?

Reply:
I think it's okay to have this migration script, but before we deploy this latest batch of work, we should turn off the feature flag, and only enable it once everything is migrated and ready. Just in case.